src/main/java/org/commcare/formplayer/installers/FormplayerLocaleInstaller.java

```diff
@@ -0,0 +1,36 @@
+package org.commcare.formplayer.installers;
+
+import org.commcare.resources.ResourceInstallContext;
+import org.commcare.resources.model.*;
+import org.commcare.resources.model.installers.LocaleFileInstaller;
+import org.commcare.util.CommCarePlatform;
+import org.javarosa.core.reference.Reference;
+
+import java.io.IOException;
+
+/**
+ * ResourceInstaller for locale files that always persists the locale data in
+ * the SQLite DB instead of referencing it from the application archive file.
+ */
+public class FormplayerLocaleInstaller extends LocaleFileInstaller {
+
+    /**
+     * Serialization only!
+     */
+    @SuppressWarnings("unused")
+    public FormplayerLocaleInstaller() {
+    }
+
+    public FormplayerLocaleInstaller(String locale) {
+        super(locale);
+    }
+
+    @Override
+    public boolean install(Resource r, ResourceLocation location, Reference ref, ResourceTable table, CommCarePlatform platform, boolean upgrade, ResourceInstallContext resourceInstallContext) throws UnresolvedResourceException {
+        try {
+            return cache(ref.getStream(), r, table, upgrade);
```

Comment:
I think we should probably change the name to something that reflects the new expectations more clearly, like "copyToInstaller" to reflect that this is a permanent part of the record now.

Reply:
Are you thinking I should change that in the base class or in this class. The other option would be to create a completely new installer that either follow the same pattern as this or is more similar to the suite / profile installers.